src/libraries/System.Text.RegularExpressions/tests/Regex.Match.Tests.cs

```diff
@@ -1144,6 +1244,12 @@ public async Task Match_SpecialUnicodeCharacters_Invariant(RegexEngine engine)
         [MemberData(nameof(RegexHelpers.AvailableEngines_MemberData), MemberType = typeof(RegexHelpers))]
         public void Match_ExcessPrefix(RegexEngine engine)
         {
+            if (RegexHelpers.IsNonBacktracking(engine))
+            {
+                // TODO-NONBACKTRACKING: Takes too long and sometimes times out with NonBacktracking
+                throw new SkipTestException("Takes too long and sometimes times out with NonBacktracking");
```

Comment:
@veanes, @olsaarik, this test takes a very long time with NonBacktracking, so I've disabled it for now.

Reply:
fixed in #1630